manipulation_pepper/GraspServerRequest request
---
manipulation_pepper/GraspConfigList grasp_configs
